### ISSUE TITLE
Get Model's X,Y,Z values when user taps on the screen.

### ISSUE THREAD

**Issue by dilipkold** [the3deers/android-3D-model-viewer#165]
Is it possible to mark over 3D model when user taps on any part of it on the screen?

**Comment by Gravity-of-eath** [comment]
yes ! if you click the "Toggle collision" menu at top-right on the demo page;

**Comment by andresoviedo** [comment]
Hello @dilipkold . there is a new TouchEvent that should contain selected object + some metadata like xyz pos. Please let me know if you still need help.

**Comment by dilipkye** [comment]
I used collision detection to mark points as you suggested, now how is this TouchEvent’s metadata is helpful over CollectionEvent? & I also need to know about moving Camera freely, reducing Object’s Vertices/Facets.
I am Sorry for not replying to your previous help, I somehow read the email and forgot to reply.
Thank you for the help!

**Comment by andresoviedo** [comment]
TouchEvent is about raw x,y screen coordinates.
CollisionEvent is a 3d x,y,z world space coordinates
